### DISCUSSION TITLE
URL image background color

### DISCUSSION THREAD

**Discussion by agravity-philipp** [rocksdanister/lively#836] [General]
**Is your feature request related to a problem? Please describe.**

I want to use the Lively Wallpager to just simply show a picture from an URL. This works fine, but it will always make a dark background and/or does not respect the pictures transparency (PNG, WEBP).

**Describe the solution you'd like**

I want to define a background color which should be "behind" this image OR it should take the one which I have defined in Windows AND/OR respect the pictures transparency (PNG, WEBP).

**Describe alternatives you've considered**

I would have to make the picture FULL SCREEN to have it placed correctly - but then I have the problem with the different resolutions.

**Additional context**
Add any other context or screenshots/mockups about the feature request here.

Great software!

**Comment by rocksdanister** [comment]
Converted to discussion since this looks like a html wallpaper creation question and not software related.

Create a page with custom background color:
https://www.w3schools.com/tags/tag_body.asp

**Reply by agravity-philipp** [reply]
Sure - This would be a fast "fix". But since we are directing to a image directly (png, webp, jpg) we could not place there a html page. Thats why I wanted to suggest to add a custom background color to it. You obviously add "black" to fill the rest of the screen when I point to an image.

Thanks for moving and converting the issue.
